### DISCUSSION TITLE
Issue with access to USB in Proxmox LXC Container

### DISCUSSION THREAD

**Discussion by TommySharpNZ** [Koenkk/zigbee2mqtt#6419] [General]
I have a Ubuntu 18 container running and have gone through the zigbe2MQTT install guide for bare metal.

My final issue is...
`
Zigbee2MQTT:error 2021-02-22 21:55:38: Error: Error while opening serialport 'Error: Error: Permission denied, cannot open /dev/bus/usb/002/002'`

I have gone through a few tutorials on how to pass through the USB device in Proxmox and I think I have it working well, the command below does list the device but the "nobody" and "nogroup" rings alarm bells but being new to Linux I'm not sure how to resolve this...

```
root@pveZigbee2MQTT:/opt/zigbee2mqtt# ls -l /dev/bus/usb/002/002
crw-rw-r-- 1 nobody nogroup 189, 129 Feb 22 20:39 /dev/bus/usb/002/002
```

Any help would be appreciated!

**Comment by thiscantbeserious** [comment]
You need to to set the proper permissions and forward it obviously so that the container can properly access it. Whiskerz is using hassio within lxc on proxmox but you can use this script for your purpose also most likely, which will take care of mounting them automatically with the proper permissions.

If you have any questions please direct it to him (in his repository).

https://github.com/whiskerz007/proxmox_hassio_lxc/blob/master/set_autodev_hook.sh

See also:

https://github.com/whiskerz007/proxmox_hassio_lxc#update-device-hooks

(be careful just running this standalone without modification might do nothing since it stores the script inside variables to be used from within another script)
